src/main/java/org/apache/sling/event/impl/EventingThreadPool.java

```diff
@@ -53,11 +55,19 @@ public class EventingThreadPool implements ThreadPool {
     public @interface Config {
 
         @AttributeDefinition(name = "Pool Size",
-              description="The size of the thread pool. This pool is used to execute jobs and therefore "
-                        + "limits the maximum number of jobs executed in parallel.")
-        int minPoolSize() default 35;
+                description = "The size of the thread pool. This pool is used to execute jobs and therefore "
+                        + "limits the maximum number of jobs executed in parallel. "
+                        + "A value of -1 is substituted with the number of available processors. "
+                        + "A decimal number between 0.0 and 1.0 is treated as a fraction of available processors. "
+                        + "For example 0.5 means half of the available processors. ")
+        double minPoolSize() default 35;
     }
 
+    /**
+     * Logger.
+     */
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
I think that in most Sling classes we use a static Logger:

```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(EventingThreadPool.class);
```

Reply:
Done